ipc/cli/src/commands/subnet/join.rs

```diff
@@ -37,7 +43,7 @@ impl CommandLineHandler for JoinSubnet {
                 public_key,
             )
             .await?;
-        log::info!("joined at epoch: {epoch}");
+        println!("joined at epoch: {epoch}");
```

Comment:
Why stop using `log`?

Reply:
Mikhail suggested that command output should ascribe to the Unix UX recommendations. That means that command outputs shouldn't return logs but raw prints so they can be piped and processed easily.